packages/design-system/src/components/contextMenu/contextMenu.js

```diff
@@ -26,7 +26,13 @@ import Menu, { MenuPropTypes } from './menu';
 import AnimationContainer from './animationContainer';
 import Mask from './mask';
 
-const ContextMenu = ({ animate, isAlwaysVisible, items, ...props }) => {
+const ContextMenu = ({
+  animate,
+  isAlwaysVisible,
+  items,
+  isInline = false,
```

Comment:
@BrittanyIRL Since the Page buttons and the element quick-action menu now share one wrapper and move together while sharing the same space, it needed some adjustments.

Let me know if you have any concerns with this.
The related changes:
- `isInline` prop that uses `relative` position instead of `absolute` for the `ContextMenu`
- `QuickActionsArea` is not a separate area anymore and is part of `PageMenuArea` that includes both the quick actions menu and the Page Menu buttons.

Reply:
@samwhale  can you take a look at this update since you've been in the trenches on this recently?

Reply:
This looks good for the Popover - but we should make sure `isInline` applies the same way (or is ignored) by the `AnimationContainer` too 😄

Reply:
Thanks for your input, will do!